osgpkitools/utils.py

```diff
@@ -140,35 +140,27 @@ def check_permissions(path):
         raise FileWriteException("User does not have appropriate permissions for writing to " + path)
 
 
-def find_user_cred(usercert=None, userkey=None):
-    """Find a readable user cert/key pair, trying pairs in the following order:
-    1. usercert, userkey
-    2. X509_USER_CERT, X509_USER_KEY environment variables
-    3. '~/.globus/usercert.pem', '~/.globus/userkey.pem'
-    INPUT (optional)
-    usercert: path to user certificate
-    userkey: path to private key of user
-    OUTPUT
-    Paths to the user cert and key
+def verify_user_cred(usercert, userkey):
+    """Verify the  readable user cert/key pair
+    INPUT
+        usercert: path to user certificate 
+        userkey: path to private key of user 
+    OUTPUT 
+        Paths to the verified user cert and key 
     """
-    
-    # list of cert/key pairs to try
-    input_pairs = [(usercert, userkey),
-                   ((os.environ.get('X509_USER_CERT'), os.environ.get('X509_USER_KEY'))),
-                   (os.path.expanduser('~/.globus/usercert.pem'), os.path.expanduser('~/.globus/userkey.pem'))]
-    cert_key_pairs = [t for t in input_pairs if None not in t] # remove undefined pairs for an improved err msg below
 
+    cert = os.path.expanduser(usercert)
+    key = os.path.expanduser(userkey)
```

Comment:
Now that there's no fallback, I don't actually see any need for the expanduser calls.  Can you think of any example cases where it makes a difference?

Reply:
The case that Joe B. reported with long option, equal sign "=" and the path starting with tilde:

`osg-incommon-cert-request -T -u fnalclient@fnal.gov --cert=~certs/jteheran_cert.pem --pkey=~/certs/jteheran_key.pem`

Path cannot be found if I don't do the expanduser(path)

Reply:
Oh I was thinking there wasn't supposed to be an equal there but I got it confused with the short version, a separate issue, ok.